src/app/modules/item/pages/item-log-view/item-log-view.component.html

```diff
@@ -1,14 +1,22 @@
 <ng-container *ngIf="state$ | async as state">
-  <p-progressSpinner *ngIf="state.isFetching" [style]="{width: '3.3333rem', height: '3.3333rem'}"></p-progressSpinner>
+  <p-progressSpinner *ngIf="state.isFetching && !state.data" [style]="{width: '3.3333rem', height: '3.3333rem'}"></p-progressSpinner>
 
   <p *ngIf="state.isError" class="alg-error-message">
     <i class="fa fa-exclamation-triangle"></i>
     <span i18n>Unable to load the recent activity</span>
+    <p-button icon="pi pi-refresh" (click)="refresh()" styleClass="p-button-sm"></p-button>
   </p>
 
-  <ng-container *ngIf="state.isReady">
-    <p-table class="alg-table" [columns]="state.data.columns" [value]="state.data.rowData">
+  <ng-container *ngIf="state.data">
+    <p-table class="alg-table" [columns]="state.data.columns" [value]="state.data.rowData" [loading]="state.isFetching">
```

Comment:
The "empty" case is not handled in this table?

Reply:
The empty state looks like this:
![image](https://user-images.githubusercontent.com/16257159/126122080-0b56bb56-5d8d-42a1-b3de-760f3e9bdbf1.png)